src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -76,6 +76,9 @@ internal sealed class VisualStudioProject
         // Effective boolean value to determine if analyzers should be executed based on _runAnalyzersPropertyValue and _runAnalyzersDuringLiveAnalysisPropertyValue.
         private bool _runAnalyzers = true;
 
+        // Stores the F# command-line options that come from the project system, which includes ordered source files that the F# language service needs.
```

Comment:
We provided the API for file ordering already, so what is still being used here that isn't expressed elsewhere? My motivation here is just to make sure the comment captures what this is still needed for and also to refresh my memory.

Reply:
We want our tooling to flow 100% from Roslyn workspaces. However, we currently rely on a MEF hack from CPS in order to get the command line options string (equivalent to C# compilation options before parsed). It contains compiler flags such as language versions, warning flags, etc. I believe we need this if we are ever going to work off of remote workspaces.